synthesizer/src/vm/execute.rs

```diff
@@ -187,13 +187,13 @@ mod tests {
 
         // Assert the size of the transaction.
         let transaction_size_in_bytes = transaction.to_bytes_le().unwrap().len();
-        assert_eq!(1387, transaction_size_in_bytes, "Update me if serialization has changed");
+        assert_eq!(1468, transaction_size_in_bytes, "Update me if serialization has changed");
```

Comment:
Why has the proof size gone up by this much? At most it should be up by 16 bytes (we removed one commitment (+ `h_0`, - `z_a` and `z_b`), and added two field elements (- z_b, + sum_a, sum_b, sum_c)